README.md

```diff
@@ -28,20 +20,530 @@ data portals
 - 🚀 NextJS framework: so everything in NextJS for free React, SSR, static site generation, huge number of examples and integrations etc.
   - SSR => unlimited number of pages, SEO etc whilst still using React.
   - Static Site Generation (SSG) (good for small sites) => ultra-simple deployment, great performance and lighthouse scores etc
-- 📋 Typescript support
```

Comment:
Any reason to remove this?

Reply:
We are currently using plain JS for building the components. The ckan examples used Typescript original, probably that's why it was there.